src/seedu/addressbook/commands/AddCommand.java

```diff
@@ -15,7 +19,7 @@
 import seedu.addressbook.data.tag.UniqueTagList;
 
 /**
- * Adds a person to the address book.
+ * Adds a person to the address book with name in alphabetical order.
```

Comment:
The person list was originally sorted by chronological order. It might be easier for the users to find a person in the address book. In this case, is it better to create a sort command to let the user decide which order they prefer to list the persons?